api/api_autogen/library/defaults/Equpartflip_GeothermalPowerAllEquityPartnershipFlip.json

```diff
@@ -234,7 +234,7 @@
 			"rate_escalation": [0.000000]
 		},
 		"Recapitalization": {
-			"system_recapitalization_cost": 2.82098e+07,
+			"system_recapitalization_cost": 8.68415e+07,
 			"system_use_recapitalization": 1
```

Comment:
Wow, a 4x increase in recapitalization for Geothermal - is this from the GEOVision study?

Reply:
@mjprilliman Any insights on why this would change? This was updated automatically by https://github.com/NREL/SAM/blob/develop/deploy/overwrite_all_defaults.lk